packages/block-editor/src/components/iframe/index.js

```diff
@@ -62,6 +62,9 @@ function bubbleEvents( doc ) {
 			init.clientY += rect.top;
 		}
 
+		// This stopPropagation call ensures React doesn't create a syncthetic event to bubble this event
+		// which would result in two React events being bubbled throught the iframe.
+		event.stopPropagation();
 		const newEvent = new Constructor( event.type, init );
 		const cancelled = ! frameElement.dispatchEvent( newEvent );
```

Comment:
You can probably avoid the stopPropagation if you dispatch the event on `frameElement.ownerDocument.body` instead. Looking through the code that should work for all the bubbled events (all either listening to body, document, or window).

Reply:
yes, but because of the scoping in place we want the event to actually go through all the parents of the iframe and not just the body.